sdk/mediaservices/test-resources.json

```diff
@@ -0,0 +1,75 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
```

Comment:
We are trying to have all the newer templates use bicep files so as you onboard can you please use a bicep template instead of json arm template.

Reply:
You can convert the ARM template you already have to bicep by running `az bicep decompile -f test-resources.json`.

More docs on bicep [here](https://docs.microsoft.com/en-us/azure/azure-resource-manager/bicep/overview?tabs=bicep).
The github also has a directory of great examples [here](https://github.com/Azure/bicep/tree/main/docs/examples).